recipes/geomatics/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "geomatics" %}
+{% set version = "0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 695bef61dad70fe1d9edbaccca99f333fb8eb4c635f614cae6fa0363744969bb
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - rasterio
+    - rasterstats
+    - netCDF4
+    - requests
+    - python-dateutil
+    - numpy
+    - pandas
+    - pyshp
+    - pygrib
+
+test:
+  imports:
+    - rasterio
+    - rasterstats
+    - netCDF4
+    - requests
+    - dateutil
+    - numpy
+    - pandas
+    - shapefile
+    - pygrib
```

You should only import things which this package provides, presumably:
```suggestion
    - geomatics
```